specs/FamilySafety.md

```diff
@@ -0,0 +1,62 @@
+Family Safety
+===
+
+# Background
+Provide end evelolper a new API to toggle Family Safety feature on and off. Once Family Safety is enabled, developer won't be able to turn it off while webview2 instance is running. Once enable, it will provide the same functionally as the browser like: Activity report, Safe Search and Web Filtering. Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void WebView_ProcessInfosChanged(object sender, object e)
```

Comment:
The C# & C++ code should match. Putting this in the process info changed event handler seems wrong?